core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -364,6 +364,11 @@ public enum Property {
       PropertyType.TIMEDURATION,
       "Time to wait between scanning tablet states to identify tablets that need to be assigned, un-assigned, migrated, etc.",
       "2.1.2"),
+  MANAGER_TABLET_GROUP_WATCHER_SCAN_THREADS("manager.tablet.watcher.scan.threads.max", "16",
```

Comment:
Should the default be 8 as thats what it was before? If not this will change the current behavior

Reply:
yeah that would be good to keep it at 8, pulled 16 from the 4.0 code.